django/db/migrations/operations/models.py

```diff
@@ -12,6 +12,16 @@
 )
 
 
+def _check_for_duplicates(arg_name, objs):
+    used_vals = set()
+    for val in objs:
+        if val in used_vals:
+            raise ValueError(
+                'Found duplicate %s in CreateModel operation.' % arg_name
```

Comment:
It was in the initial version of patch when bases and managers were not also checked. If you think this must be present I can submit a PR with the previous changes.